doc/internal/man3/ossl_provider_new.pod

```diff
@@ -208,6 +213,12 @@ responder.
 It should treat the given I<OSSL_PARAM> array as described in
 L<OSSL_PARAM(3)>.
 
+ossl_provider_get_capabilties() calls the provider's I<get_capabilties> function,
```

Comment:
NIT: capabilties ?? (this is done on the next line also)

Reply:
I think there's a problem with my keyboard. It can't seem to spell that word. I fixed it.